pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.haml

```diff
@@ -61,7 +61,7 @@ theme: responsive_full_width
           %li No computer science experience required
           %li Based in United States
           %li Teaching 25+ hours of course to a class in next school year
-        %a{href: CDO.studio_url("/pd/application/teacher") , class: "link-button", target: "_blank", rel: "noopener noreferrer"} Apply now
+        %a{href: "/educate/professional-learning/program-information" , class: "link-button", target: "_blank", rel: "noopener noreferrer"} Apply now
```

Comment:
Nit but not blocking: there's a space between the closing quote and the comma `" ,`